src/main/java/whackamole/whackamole/Main.java

```diff
@@ -1,48 +1,48 @@
 package whackamole.whackamole;
 
-import dev.jorel.commandapi.*;
-import org.bukkit.plugin.Plugin;
-import org.bukkit.plugin.java.JavaPlugin;
-
+import dev.jorel.commandapi.CommandAPI;
+import dev.jorel.commandapi.CommandAPIConfig;
+import whackamole.whackamole.DB.SQLite;
 
+import org.bukkit.plugin.java.JavaPlugin;
 
 public final class Main extends JavaPlugin {
-
-    private Translator translator = Translator.getInstance();
-    private Logger logger = Logger.getInstance();
-    private Commands commands;
-    public GamesManager manager;
-    public static Plugin plugin;
+    public GamesManager manager = GamesManager.getInstance();
 
     @Override
     public void onLoad() {
-        Logger.Prefix = this.getDescription().getPrefix();
-        Config.getInstance(this);
         CommandAPI.onLoad(new CommandAPIConfig().verboseOutput(false));
-        Main.plugin = this;
+
+        Logger.onLoad(this);
+        if (!Config.onLoad(this)) {
```

Comment:
@CraZyRc If u agree to handle the config loading failing this way, will u then add a translation for this?